infra/build-services.md

```diff
@@ -0,0 +1,36 @@
+# Choice of Build Pipelines
+
+Currently, our builds run on [Azure Dev Ops Pipelines](https://dev.azure.com/responsibleai/fairlearn/_build) (hereafter ADO).
```

Comment:
GitHub & Azure are both owned by Microsoft, so I would see the "good open source citizen" gain as minimal. The real benefit comes from direct integration in the GitHub UI. No more clicking around, suddenly landing in Azure Pipelines (different UI), and then having to navigate back to GitHub.

Reply:
I've added that.